pkgs/tools/security/chipsec/default.nix

```diff
@@ -11,28 +11,20 @@
 
 python3.pkgs.buildPythonApplication rec {
   pname = "chipsec";
-  version = "1.10.6";
-  format = "setuptools";
+  version = "1.13.17";
+
+  pyproject = true;
+  build-system = [ python3.pkgs.setuptools ];
```

Comment:
It's probably be better to move this near `dependencies`, so all the inputs are grouped together.